nemo/collections/asr/parts/utils/speaker_utils.py

```diff
@@ -611,6 +613,29 @@ def isOverlap(rangeA, rangeB):
     return end1 > start2 and end2 > start1
 
 
+def validate_vad_manifest(AUDIO_RTTM_MAP, vad_manifest):
+    """
+    This function will validate the manifest file from NeMo voice activity detection or Oracle VAD to the uniq_ids present 
+    in provided audio manifest file. If there is a mis-match we ignore those uniq_ids from clustering.
```

Comment:
Suggest to rephrase like this: 

`This function will check the valid speech segments in the manifest file which is either generated from NeMo voice activity detection (VAD) or oracle VAD. If the audio file does not contain any valid speech segments, we ignore the audio files (indexed by uniq_id) for the rest of the processing steps.`